src/App.tsx

```diff
@@ -97,6 +97,7 @@ export class App extends React.Component<{}, {
                 <Route path="/daos" component={AppContainer}/>
                 <Route path="/profile" component={AppContainer}/>
                 <Route path="/redemptions" component={AppContainer}/>
+                <Route path="/dao-creator" component={AppContainer} />
```

Comment:
I've changed it to `/daos/create` so that it doesn't conflict with the `/dao/${address}` route. `/daos/create` follows the same pattern as `proposals/create`.